src/Pages/Home/Home.js

```diff
@@ -1,6 +1,75 @@
 import "./Home.css";
+import { ShoetubeContainer } from "../../components/Wrapper/ShoetubeContainer";
+import { Chip } from "../../components/Chip/Chip";
+import { SearchBar } from "../../components/SearchBar/SearchBar";
+import { VideoCard } from "../../components/Cards/VideoCard/VideoCard";
+import { useCategory } from "../../context/categoryContext";
+import { useVideos } from "../../context/videoContext";
+import { getFinalData } from "../../utils/videos/getFinalData";
+import { useState } from "react";
+import { usePlaylist } from "../../context/playlistContext";
+
 const Home = () => {
-  return <div>Home</div>;
+  const [searchInput, setSearchInput] = useState("");
+  const { categoryState, selectedCategory, setSelectedCategory } =
+    useCategory();
+  const { videoState } = useVideos();
+  const { playlistState } = usePlaylist();
+  const finalData = getFinalData(videoState, selectedCategory, searchInput);
+  console.log(playlistState);
+  return (
+    <ShoetubeContainer>
+      <div className="flex-col bord-3-green">
+        <div className="chip-section bord-3-purple flex-row align-center-flex gap-1">
+          <Chip
+            style={{
+              backgroundColor: selectedCategory === "All" ? "black" : "",
```

Comment:
would recommend setting a class here instead of inline styles.
suggestion: send the classname to the component